src/clients/c++/examples/simple_sequence_client.cc

```diff
@@ -276,47 +275,48 @@ main(int argc, char** argv)
   std::vector<int32_t> result1_list;
 
   std::vector<std::unique_ptr<nic::InferContext>> ctxs;
```

Comment:
You can move this declaration down as well?

Reply:
Done